tests/Unit/Time/Test_Slab.cpp

```diff
@@ -9,7 +9,7 @@
 #include "tests/Unit/TestHelpers.hpp"
 
 SPECTRE_TEST_CASE("Unit.Time.Slab", "[Unit][Time]") {
-  Approx approx =
+  Approx my_approx =
       Approx::custom().epsilon(5 * std::numeric_limits<double>::epsilon());
```

Comment:
@wthrowe is there a good reason (after the change by this PR) why 5 is needed instead of the 100 the new approx defaults to?  If so, let's comment the reason; if not I'd suggest going with the default...